### PR TITLE
Documentation!!

### PR DESCRIPTION
**Pull request by Islast**
- [x] I'm ready to merge

* **What's the context for this pull request?** 
_(this is a good place to reference any issues that this PR addresses)_
#76 and #45

* **What's new?**
    * docstrings for all functions and classes (except make_figures, which we haven't refactored yet)
    * doc folder has been renamed docs to allow compatibility with project pages.
    * documentation should be ready to deploy to site.
    * a guide for developers at the end of the contributing guidelines

* **What should a reviewer feedback on?**
the style of documentation or missing documentation
It would be wonderful if Kirstie could look over the developer guidelines and check it all makes sense. It's actually a bit of a first draft.

* **Does anything need to be updated after merge?** 
_(e.g the wiki or the WhitakerLab website)
When this is merged into master (and sadly not before) we will be able to edit the GitHub pages settings of this repository and deploy the docs folder to github pages

### CODE REVIEW

**Comment by Islast** [comment]
Thank you so much @rmarkello I'm glad you like it! :sun_with_face:  Yes, you are welcome to adapt it for your own projects, and a link like that back to BrainNetworksInPython will be more than enough :+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -164,3 +166,200 @@ And if you've found typos in this (or any other) page, you could consider submit
 [link_hatrackidea]: http://hawthornlandings.org/2015/02/13/a-place-to-hang-your-hat/
 [link_hatrackcontributions]: http://opensource.com/life/15/10/octohat-github-non-code-contribution-tracker
 [link_fixingtyposissue]: https://github.com/WhitakerLab/BrainNetworksInPython//issues/4
+
+
+# Development Guide
+
```
[the remark this answers is not in the document]
Comment (KirstieJane) [reply CONTRIBUTING.md]: DISREGARD. I've changed my mind.

Keep it as a main heading, and add just a couple of sentences about the fact that this is a detailed set of instructions.

If you get *really* excited you could link to it from higher up in the file, but don't worry about that for now as @KirstieJane (mee) is going to edit some of the earlier docs based on the BIDS Starter Kit guidelines.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -164,3 +166,200 @@ And if you've found typos in this (or any other) page, you could consider submit
...
+        assert bnip.calc_leaving_module_interhem(self.G2, M3) == {0: 1, 1: 1}
+      def G2_modules_are_split_across_hemispheres_values_0(self):
+        # the leaving interhem values should be zero for each module, since since
+        # none of the leaving edges are interhemispheric
+        M4 = {0: {0, 1}, 1: {2, 3}}
+        assert bnip.calc_leaving_module_interhem(self.G2, M4) == {0: 0, 1: 0}
+      def G2_test_module_M5(self):
+        M5 = {0: {0}, 1: {1, 2, 3}}
+        assert bnip.calc_leaving_module_interhem(self.G2, M5) == {0: .5, 1: .5}
+  ```
```
Comment (KirstieJane) [inline CONTRIBUTING.md]: Just add a little thank you at the end and a link to how to ask questions 😸

**Comment by KirstieJane** [comment]
I'm doing it! We can develop further in the WhitakerLab dev branch!! 😱 ✨ 🙌

**Pull request merged**